index.html

```diff
@@ -11,6 +11,11 @@
     <title>Portfolio home page</title>
   </head>
 <body>
+
+<!--detail popup page -->
+<div class="popup-background"></div>
+<section id="card4-page" class="popup"></section>
+
 <!-- Main page section -->
 <section id="main-page">
   <header>
```

Comment:
- [ OPTIONAL ] I noticed that the desktop and mobile views of your header section are missing the background images that were applied to them previously, this is making your project look different from the Figma Design so kindly do well to fix this as well :+1: 
- Your Design Now
![image](https://user-images.githubusercontent.com/102699958/216155289-a001362b-3c77-4475-980b-2550826a229e.png)
- The Figma Design
![image](https://user-images.githubusercontent.com/102699958/216155471-44afe1cf-de5d-4ef9-a0f2-c610b16d3b5d.png)

Reply:
Dear @NahnahAJ  Thank you for the well detailed feedback. Very helpful. However, The background picture works fine for me. 
![Screenshot from 2023-02-02 01-05-26](https://user-images.githubusercontent.com/35451409/216188562-4c0bf11e-194d-4083-9616-87209cd3407f.png)

I also got the same feedback last time and I changed the image type to png and it got accepted.